From 0875ddcb19eef3860fb7d8c88653e47319cb3e3e Mon Sep 17 00:00:00 2001
From: Andre Heider <a.heider@gmail.com>
Date: Fri, 18 Nov 2022 12:31:19 +0100
Subject: [PATCH 11/12] iwinfo: fix -Wformat-truncation warnings

Increase buffer sizes to get rid of e.g.:
iwinfo_cli.c:54:45: warning: '%d' directive output may be truncated writing between 1 and 10 bytes into a region of size 8 [-Wformat-truncation=]

Signed-off-by: Andre Heider <a.heider@gmail.com>
---
 iwinfo_cli.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

--- a/iwinfo_cli.c
+++ b/iwinfo_cli.c
@@ -46,7 +46,7 @@ static char * format_ssid(char *ssid)
 
 static char * format_channel(int ch)
 {
-	static char buf[8];
+	static char buf[16];
 
 	if (ch <= 0)
 		snprintf(buf, sizeof(buf), "unknown");
@@ -70,7 +70,7 @@ static char * format_frequency(int freq)
 
 static char * format_txpower(int pwr)
 {
-	static char buf[10];
+	static char buf[16];
 
 	if (pwr < 0)
 		snprintf(buf, sizeof(buf), "unknown");
@@ -82,7 +82,7 @@ static char * format_txpower(int pwr)
 
 static char * format_quality(int qual)
 {
-	static char buf[8];
+	static char buf[16];
 
 	if (qual < 0)
 		snprintf(buf, sizeof(buf), "unknown");
@@ -94,7 +94,7 @@ static char * format_quality(int qual)
 
 static char * format_quality_max(int qmax)
 {
-	static char buf[8];
+	static char buf[16];
 
 	if (qmax < 0)
 		snprintf(buf, sizeof(buf), "unknown");
